packages/relay/tests/lib/utils.spec.ts

```diff
@@ -45,4 +46,39 @@ describe('Utils', () => {
       });
     }
   });
+
+  describe('createSyntheticLogsBloom and checkInLogsBloom', () => {
+    const address = '0x000000000000000000000000000000000000040c';
+    const topics = [
+      '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef',
+      '0x00000000000000000000000000000000000000000000000000000000000003f5',
+      '0x00000000000000000000000000000000000000000000000000000000000003f6',
+    ];
+    const expectedLogsBloom =
+      '0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000' +
+      '000000000000000000000000000000000000000000000000001000000000000000800000000000000000000000000000000000000000000' +
+      '000000000000000000000000000000000000000000000000001000000000000000000400000000000000000000000000000000000000000' +
+      '000000000000100000000000000000000000000000000000000000000000000000400000000000000000300000000040000000000000000' +
+      '0000000000000000000000080000000000001000000000000000000000000000000000000000000000000001000000';
+
+    it('should be able to generate logsBloom of transfer event', () => {
+      const res = Utils.createSyntheticLogsBloom(address, topics);
+      expect(expectedLogsBloom).to.equal(res);
+    });
+
+    it('should be able to validate address and topics in generated logsBloom', () => {
+      expect(Utils.checkInLogsBloom(address, expectedLogsBloom)).to.equal(true);
+      expect(Utils.checkInLogsBloom(topics[0], expectedLogsBloom)).to.equal(true);
+      expect(Utils.checkInLogsBloom(topics[1], expectedLogsBloom)).to.equal(true);
+      expect(Utils.checkInLogsBloom(topics[2], expectedLogsBloom)).to.equal(true);
+    });
+
+    it('should be able to validate non-existing address and topic in generated logsBloom', () => {
```

Comment:
Do we have a test for the scenario where an emptyBloom is returned, so basically testing the first if in the _createSyntheticLogsBloom_ method?

Reply:
Added.